SharpMUSH.Client/Components/WikiDisplay.razor.css

```
@@ -0,0 +1,146 @@
+    border-left: 2px solid transparent;
+    transition: color 0.12s, border-color 0.12s;
+}
+.wiki-toc-list a:hover {
+    color: var(--accent);
+    border-left-color: var(--accent);
+}
+
+/* Mermaid diagrams (rendered client-side by js/wiki-render.js). Before mermaid
+   processes the block it is a pre.mermaid of raw source; after, it holds an SVG.
+   Center it and let the SVG scale down on narrow viewports. */
+.WikiContent ::deep .mermaid {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)